extras/AFC.py

```diff
@@ -256,7 +260,11 @@ def handle_connect(self):
         except:
             self.bypass = add_filament_switch("filament_switch_sensor virtual_bypass", "afc_virtual_bypass:virtual_bypass", self.printer ).runout_helper
 
+        if self.show_quiet_mode:
+            self.quiet_switch = add_filament_switch("filament_switch_sensor virtual_quiet_mode", "afc_quiet_mode:afc_quiet_mode", self.printer ).runout_helper
```

Comment:
Lets just remove the `virtual` part and just name the switch `quiet_mode`
```suggestion
            self.quiet_switch = add_filament_switch("filament_switch_sensor quiet_mode", "afc_quiet_mode:afc_quiet_mode", self.printer ).runout_helper
```